earthfile2llb/interpreter.go

```diff
@@ -125,7 +125,8 @@ func (i *Interpreter) handleBlock(ctx context.Context, b spec.Block) error {
 		if err != nil {
 			return err
 		}
-		prevWasArg = (stmt.Command != nil && stmt.Command.Name == "ARG")
+		prevWasArg = stmt.Command != nil && (stmt.Command.Name == "ARG" || (i.converter.ftrs.LetSetBlockParallel && (stmt.Command.Name == "LET" || stmt.Command.Name == "SET")))
```

Comment:
I find these long conditional expressions difficult to mentally parse. How about a function with guards instead?

```golang
func (i *interpreter) prevWasArg(stmt *Statement) bool {
	if (stmt == nil || stmt.Command == nil) {
		return false
	}

	if i.converter.ftrs.LetSetBlockParallel && (stmt.Command.Name == "LET" || stmt.Command.Name == "SET") {
		return true
	}

	return stmt.Command.Name == "ARG"
}
```